applications/MeshingApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -47,11 +47,40 @@ namespace Python
 {
 namespace py = pybind11;
 
+void RecursiveEnsureModelPartOwnsProperties(ModelPart& rModelPart)
+{
+    MeshingUtilities::RecursiveEnsureModelPartOwnsProperties(rModelPart);
+}
+
+void RecursiveEnsureModelPartOwnsPropertiesWithClear(
+    ModelPart& rModelPart,
+    const bool RemovePreviousProperties
+    )
+{
+    MeshingUtilities::RecursiveEnsureModelPartOwnsProperties(rModelPart, RemovePreviousProperties);
+}
+
+void EnsureModelPartOwnsProperties(ModelPart& rModelPart)
+{
+    MeshingUtilities::EnsureModelPartOwnsProperties(rModelPart);
+}
+
+void EnsureModelPartOwnsPropertiesWithClear(
```

Comment:
I was discussing this week with jordi, maybe in the future we should start using fct pointers, would clean up a lot

Reply:
fct pointers¿?

Reply:
E.g. like this: https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python/add_data_communicator_to_python.cpp#L77
Then we don’t need the aux function